webhooks/common.go

```diff
@@ -0,0 +1,55 @@
+package webhooks
+
+import (
+	"fmt"
+	"strings"
+
+	api "github.com/open-feature/open-feature-operator/apis/core/v1beta1"
+	"github.com/open-feature/open-feature-operator/common/constant"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func OpenFeatureEnabledAnnotationIndex(o client.Object) []string {
+	pod := o.(*corev1.Pod)
```

Comment:
just to be safe, we can do a check if the casting has been successful to prevent panics if `o` is `nil`
```suggestion
	pod, ok := o.(*corev1.Pod)
	if !ok {
		return []string{"false"}
	}
```